interceptors/auth/metadata.go

```diff
@@ -7,10 +7,10 @@ import (
 	"context"
 	"strings"
 
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/metadata"
```

Comment:
I can see you moved all of these imports around, I think we want to move them down again.

Reply:
Fixed.